server/tests/test_parse_android.json

```diff
@@ -0,0 +1,136 @@
+[{
```

Comment:
Where is this used? Should this be merged with `tests/inputs/test_parse_android.json`